auditor/src/routes/add.rs

```diff
@@ -182,6 +182,138 @@ pub async fn add_record(record: &RecordAdd, pool: &PgPool) -> Result<(), AddReco
     }
 }
 
+#[tracing::instrument(name = "Adding a record to the database", skip(records, pool))]
+pub async fn bulk_add(
```

Comment:
The tracing instrument name doesn't match what the function is doing. Maybe switch to "Adding multiple records to the database" or something like this.